sklearn/tests/test_discriminant_analysis.py

```diff
@@ -199,7 +204,10 @@ def test_lda_priors():
     priors = np.array([0.5, -0.5])
     clf = LinearDiscriminantAnalysis(priors=priors)
     msg = "priors must be non-negative"
-    assert_raise_message(ValueError, msg, clf.fit, X, y)
+
+    with pytest.raises(ValueError,
+                       match=msg):
+        clf.fit(X, y)
```

Comment:
nitpick: let's use a single line when it fits in less than 80 columns.

```suggestion
    with pytest.raises(ValueError, match=msg):
        clf.fit(X, y)
```

Reply:
Done!